examples/utils/oauth.js

```diff
@@ -0,0 +1,21 @@
+// Authenticate with PureCloud
+function authenticate(client, pcEnvironment) {
```

Comment:
We could alternatively just pull the platform client instance from the window (i.e. `window.require('platformClient')`, but different examples with setting different configurations on the client instance and it felt a little magical to just pull the same instance down from the window and this is more explicit. I'm open to going this route tho as we would probably be able to eliminate a tiny bit of duplication.

Reply:
I think you made the right call.  I'll always favor explict over magical.  Also, I think this is the right balance of showing some config but hidding other nitty gritty details like auth in a util to make the demo code easier to read.